packages/manager/modules/nutanix/src/constants.js

```diff
@@ -1,3 +1,33 @@
+export const PRODUCT_ID = 'nutanix';
+export const DEFAULT_OS_NODE_NUTANIX = 'none_64.en';
+
+export const ORDER_EXPRESS_URLS = {
+  CZ: 'https://www.ovh.cz/order/express/#/new/express/resume?products=',
+  DE: 'https://www.ovh.de/order/express/#/new/express/resume?products=',
+  ES: 'https://www.ovh.es/order/express/#/new/express/resume?products=',
+  FI: 'https://www.ovh-hosting.fi/order/express/#/new/express/resume?products=',
+  FR: 'https://www.ovh.com/fr/order/express/#/new/express/resume?products=',
+  GB: 'https://www.ovh.co.uk/order/express/#/new/express/resume?products=',
+  IE: 'https://www.ovh.ie/order/express/#/new/express/resume?products=',
+  IT: 'https://www.ovh.it/order/express/#/new/express/resume?products=',
+  LT: 'https://www.ovh.lt/order/express/#/new/express/resume?products=',
+  MA: 'https://www.ovh.ma/order/express/#/new/express/resume?products=',
+  NL: 'https://www.ovh.nl/order/express/#/new/express/resume?products=',
+  PL: 'https://www.ovh.pl/order/express/#/new/express/resume?products=',
+  PT: 'https://www.ovh.pt/order/express/#/new/express/resume?products=',
+  SN: 'https://www.ovh.sn/order/express/#/new/express/resume?products=',
+  TN: 'https://www.ovh.com/tn/order/express/#/new/express/resume?products=',
+  US: 'https://us.ovhcloud.com/order/express/#/express/review?products=',
+  ASIA: 'https://ca.ovh.com/asia/order/express/#/new/express/resume?products=',
+  IN: 'https://ca.ovh.com/in/order/express/#/express/review?products=',
+  AU: 'https://ca.ovh.com/au/order/express/#/new/express/resume?products=',
+  CA: 'https://ca.ovh.com/en/order/express/#/new/express/resume?products=',
+  QC: 'https://ca.ovh.com/fr/order/express/#/new/express/resume?products=',
+  SG: 'https://ca.ovh.com/sg/order/express/#/new/express/resume?products=',
+  WE: 'https://us.ovh.com/us/order/express/#/new/express/resume?products=',
+  WS: 'https://us.ovh.com/es/order/express/#/new/express/resume?products=',
+};
+
```

Comment:
Why not use express order constants defined here https://github.com/ovh/manager/blob/master/packages/manager/apps/dedicated/client/app/config/constants.config.js ?